bindings/rust/standard/integration/Cargo.toml

```diff
@@ -36,3 +38,7 @@ http-body-util = "0.1"
 bytes = "1.8"
 hyper = "1.5"
 hyper-util = "0.1"
+
+[build-dependencies]
+# The ML-DSA tests require the ML-DSA support added in Openssl-3.5
+openssl-src = { version = "300.5", optional = true, features = ["legacy"] }
```

Comment:
why would you need legacy features if you're testing mldsa?

Reply:
I expanded the comment-- see if that answers your question.